contacts24/contacts.py

```diff
@@ -33,7 +34,7 @@ def parse_input(user_input: str) -> tuple[str, Optional[CommandArguments]]:
 
 def load_contacts_book() -> AddressBook:
```

Comment:
@krisdzhus  in `address_book.py` in method `load_contacts_book` or `save_to_file` we provide file as parameter.

Could we do it here, to have the same style everywhere?